pandas/tests/series/test_apply.py

```diff
@@ -124,7 +124,7 @@ def test_apply_datetimetz(self):
 
         # change dtype
         result = s.apply(lambda x: x.hour)
-        exp = pd.Series(list(range(24)) + [0], name='XX', dtype=np.int32)
+        exp = pd.Series(list(range(24)) + [0], name='XX', dtype=np.int64)
```

Comment:
this is actually an implementation detail (as they r stored as int32)

we can change but that should be separate PR

ideally just like to preserve here